packaging/datadog-agent/win32/build.ps1

```diff
@@ -39,7 +39,7 @@ cp -R ..\..\..\dist\gohai install_files\files\gohai
 cp -R install_files\guidata install_files\files
 
 # Copy the license file
-cp ..\..\..\LICENSE install_files\license.rtf
+cp ..\..\..\LICENSE.rtf install_files\license.rtf
```

Comment:
It's correct; because earlier in the script we changed directories several levels down (line 22)
		 cd packaging\datadog-agent\win32